### PR TITLE
Backend for Amenity Details (Suyash)

### PR DESCRIPTION
**Pull request by TheHarbinger99**
Hey , I have updated the django models according to expected data ingest in comments , and created the Api's for the Amenity detaills. Pushing it to develop so that Adnan and Dinesh can fork this and continue with their tasks.

### CODE REVIEW

**Review by AdnanAhsanNYCity — APPROVED** [review]
Seems all good to me!

File: NycBasics/models.py [hunk 1]
```diff
@@ -44,12 +94,29 @@ def __str__(self):
...
+        )
+        """
 
 
 class parking_model(models.Model):
     name = "parking"
     parking_latitude = models.FloatField(null=True)
     parking_longitude = models.FloatField(null=True)
+    """
+    parking_the_geom = models.CharField(max_length = 10000, null=True)
```
Comment (ds6332) [inline NycBasics/models.py:113]: Can the maximum size for the_geom be increased further? One of the multipolygon points has string length over 10000 characters.

**Review by ds6332 — COMMENTED** [review]
Except for the character size limit of parking.the_geom, everything else looks good.

**Comment by TheHarbinger99** [comment]
hey i have changed the Charfield limits for models in comments.

**Review by ds6332 — APPROVED** [review]
looks good.

**Review by AdnanAhsanNYCity — APPROVED** [review]
Yep good to me!

**Pull request merged**